services/QuillLMS/client/app/bundles/Staff/components/evidence/rulesAnalysis/ruleAnalysis.tsx

```diff
@@ -376,7 +376,7 @@ const RuleAnalysis = ({ match }) => {
         className="responses-table"
         columns={responseHeaders}
         data={responses}
-        defaultPageSize={responses.length < 100 ? responses.length : 100}
+        defaultPageSize={responses.length < 100 && responses.length > 0 ? responses.length : 100}
```

Comment:
Actually, on second viewing, another way of writing this is `Math.min(responses.length, 100) || 100`.  I think it's a style question as to whether that makes more sense than multiple conditions in a ternary, so up to you if you want to adjust it to be more algorithmic.

Reply:
yeah this looks cleaner, thanks for the suggestion